docs/manual/faq/_index.en.md

```diff
@@ -1,111 +1,133 @@
-Yes Contao supports [multi-domain operation](/de/layout/seitenstruktur/multidomain-betrieb/) and multilingual[ websites](/de/layout/seitenstruktur/mehrsprachige-webseiten/).
+{{% expand "Can I maintain multiple websites with Contao?" %}}
+Yes Contao supports [multi-domain operation](/en/layout/site-structure/multi-domain-operation/) and 
+multilingual [ websites](/en/layout/site-structure/multilingual-websites/).
 {{% /expand %}}
 
-{{% expand "Kann ich mit Contao »Mehrsprachige Webseiten« pflegen?" %}}
-Yes Contao supports [multilingual websites](/de/layout/seitenstruktur/mehrsprachige-webseiten/).
+{{% expand "Can I maintain multilingual websites with Contao?" %}}
+Yes Contao supports [multilingual websites](/en/layout/site-structure/multilingual-websites/).
 {{% /expand %}}
 
-{{% expand "Wie aktiviere ich den Contao Debug-Modus?" %}}
-You can activate the [Contao debug mode](/de/system/debug-modus/) via the backend.
+{{% expand "How do I enable the Contao debug mode?" %}}
+You can activate the [Contao debug mode](/en/system/debug-mode/) via the backend.
 {{% /expand %}}
 
-{{% expand "Wo finde ich weitere Contao-Ressourcen?" %}}
+{{% expand "Where can I find more Contao resources?" %}}
 You can find more Contao resources on the [project website](https://contao.org/de/netzwerk.html).
 {{% /expand %}}
 
-{{% expand "Darf ich Contao für kommerzielle Projekte verwenden?" %}}
-Yes, the [GNU Lesser General Public License](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html) (LGPL), under which Contao has been licensed since version 2.5, allows you to use the system for commercial projects. Please note that the copyright notices in the Contao files may not be removed or changed according to the license terms.
+{{% expand "May I use Contao for commercial projects?" %}}
+Yes, the [GNU Lesser General Public License](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html) (LGPL), 
+under which Contao has been licensed since version 2.5, allows you to use the system for commercial projects. 
+Please note that the copyright notices in the Contao files may not be removed or changed according to the license terms.
 {{% /expand %}}
 
+
 ## Template
 
-{{% expand "Wie kann ich alle Variablen meines Templates anzeigen?" %}}
-The information about this can be found under [Show Template Data](/de/templates/data/).
+{{% expand "How can I display all variables of my template?" %}}
+The information about this can be found under [Show Template Data](/en/layout/templates/template-data/).
 {{% /expand %}}
 
-{{% expand "Wie kann ich ein Insert-Tag in meinem Template verwenden?" %}}
-To use an [Insert-Tag](/de/artikelverwaltung/insert-tags/)`{{date}}` in your template you have to insert it over `$this->replaceInsertTags('{{date}}')`it.
+{{% expand "How can I use an insert tag in my template?" %}}
+To use an [Insert-Tag](/en/article-management/insert-tags/) `{{date}}` in your template you have to insert 
+`$this->replaceInsertTags('{{date}}')`.
```

Comment:
This is not relevant to this PR, but this FAQ entry seems weird. Why would you do that? You _don't_ have to use `$this->replaceInsertTags`.

Reply:
OK, should i better delete it?

Reply:
I think so, yes ;). Also from the German version.

Reply:
Done